src/components/index.d.ts

```diff
@@ -366,7 +366,8 @@ export declare namespace ValueProps {
           header: string
           subHeader: string
           alt?: undefined
-        })[]
```

Comment:
not sure why my prettier linted this as well 😬

Reply:
Doesn't bother me I dont think